include/swift/AST/Module.h

```diff
@@ -648,27 +648,52 @@ class ModuleDecl : public DeclContext, public TypeDecl {
   /// \sa getImportedModules
   enum class ImportFilterKind {
     /// Include imports declared with `@_exported`.
-    Public = 1 << 0,
+    Exported = 1 << 0,
     /// Include "regular" imports with no special annotation.
     Private = 1 << 1,
     /// Include imports declared with `@_implementationOnly`.
     ImplementationOnly = 1 << 2,
-    /// Include imports of SPIs declared with `@_spi`
+    /// Include imports of SPIs declared with `@_spi`. Non-SPI imports are
+    /// included whether or not this flag is specified.
     SPIAccessControl = 1 << 3,
-    /// Include imports shadowed by a separately-imported overlay (i.e. a
-    /// cross-import overlay). Unshadowed imports are included whether or not
-    /// this flag is specified.
-    ShadowedBySeparateOverlay = 1 << 4
+    /// Include imports shadowed by a cross-import overlay. Unshadowed imports
+    /// are included whether or not this flag is specified.
+    ShadowedByCrossImportOverlay = 1 << 4
```

Comment:
Would something like `ReplacedByCrossImport` be even clearer?

Reply:
I started implementing this and updating all the comments, but I feel like "shadowed" conveys the sense better? With shadowing, the old thing is still there, but the compiler is looking at something closer/more relevant. With "replace", it is less clear if/why the old thing sticks around.

Reply:
Sure, that's fine.